### PR TITLE
Updates all stories so they use fewer snapshots, are more succinct

### PR DESCRIPTION
**Pull request by aendrew**
This refactors the stories to be more succinct, better documented, and to use ~75% fewer snapshots. I also turned off snapshots for a few components (namely grid, VideoSet) and made snaps more deterministic.  

At this point I think we can safely use Chromatic for snapshot testing.

In the future, put all of your stories in a single story called "Examples"; you'll need to enclose it in `<Fragment>`/`<>` tags to parse properly. Create multiple stories when an example requires quite a lot of documentation to explain.

### CODE REVIEW

**Comment by tlfrd** [comment]
LGTM!

File: src/html-head/index.stories.mdx [hunk 1]
```diff
@@ -2,10 +2,8 @@ import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
...
-# x component
+# HTML header component
 
-tktk
+`HtmlHead` was originally used to get metadata into the `<head>` section of the page.
 
-<Preview>
-  <Story name="Basic"></Story>
-</Preview>
+It is no longer in use. **@TODO remove.**
```
Comment (joannaskao) [inline src/html-head/index.stories.mdx:9]: Should we just remove it now?

File: src/footer/index.stories.mdx [hunk 2]
```diff
@@ -5,10 +5,10 @@ import Footer from './';
 
 # Footer component
 
-tktk
+Basic FT.com footer. It takes no props.
 
 <Preview>
-  <Story name="Basic">
+  <Story name="Example">
```
Comment (joannaskao) [inline src/footer/index.stories.mdx:11]: Do we need to snapshot this if we're just using Origami?

File: src/expander/index.stories.mdx [hunk 3]
```diff
@@ -8,27 +8,26 @@ import Expander from './';
...
+            ))}
+        </ul>
+      </Expander>
+      Basic with multiple children:
+      <Expander>
         {Array(50)
           .fill(null)
           .map((_, idx) => (
-            <li key={idx}>{idx}</li>
+            <div key={idx}>{idx}</div>
```
Comment (joannaskao) [inline src/expander/index.stories.mdx:28]: Was this change deliberate? Why?

File: src/article-layout/index.stories.mdx [hunk 4]
```diff
@@ -45,29 +46,69 @@ export const props = {
...
-      <GridContainer>
-        <GridRow>
-          <GridChild colspan="12 S11 Scenter M10 L9 XL8">
-            <p>Lorem ipsum dolor sit amet, esse mediocritatem et eos, ex cum nostrum singulis inciderint.</p>
-            <MiddleAd />
-            <p>Ex his oratio perpetua expetendis, commune suscipiantur nec at, vel lorem atqui option ne. Habeo congue vel te.</p>
-          </GridChild>
-        </GridRow>
-      </GridContainer>
+      <main key="main" role="main">
```
Comment (joannaskao) [inline src/article-layout/index.stories.mdx:56]: Do we have to remember to add the main and article tags in every layout now?

File: src/html-head/index.stories.mdx (hunk 1 above)
Comment (aendrew) [reply src/html-head/index.stories.mdx:9]: I might resurrect it if we ever decide we want to use react-helmet-async at some point, and it's useful for historical reasons.

File: src/footer/index.stories.mdx (hunk 2 above)
Comment (aendrew) [reply src/footer/index.stories.mdx:11]: There's some value in snapshotting straight-up Origami components (e.g., without props) to ensure that nothing upstream has broken between releases, but only ever one snapshot, multiple stories are kinda overkill.

File: src/expander/index.stories.mdx (hunk 3 above)
Comment (aendrew) [reply src/expander/index.stories.mdx:28]: This isn't changing the `Basic with one child example`, this is moving the `Basic with multiple children` example up. Code hasn't changed!

File: src/article-layout/index.stories.mdx (hunk 4 above)
Comment (aendrew) [reply src/article-layout/index.stories.mdx:56]: Yes. The example given is the code being added to Starter Kit.
